mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.DeleteOrder = (req, res) => {
+  Order.remove({ _id: req.params.orderId }) 
+    .exec((error, result) => {
```

Comment:
The callback function in the `.exec()` method should have a consistent error handling approach. It's better to check if `error` is not null or undefined before checking the `result`. Also, it's a good practice to log the error for debugging purposes. 

Suggested improvement:
```
.exec((error, result) => {
  if (error) {
    console.error(error);
    return res.status(400).json({ error: errorHandler(error) });
  }
  res.json(result);
});
```